content/modules/flow-ui/pages/exception-handlers.adoc

```diff
@@ -70,4 +70,24 @@ include::example$/onboarding/src/main/java/com/company/onboarding/exception/MyEx
 <2> - You can instruct the provider to take effect only for specific exceptions. Return `true` if you want it to work for all exception types.
 <3> - Return new instance of `MyExceptionDialog`.
 
-The framework will pick up your provider and use the returned dialog for unhandled exceptions.
\ No newline at end of file
+The framework will pick up your provider and use the returned dialog for unhandled exceptions.
+
+[[router-exception-handling]]
+== Router Exception Handling
+
+Vaadin provides https://vaadin.com/docs/latest/flow/routing/exceptions[support^] for navigation target exceptions. When an unhandled exception is thrown during navigation, an error view is shown. For example, when a user navigates to a non-existent route, the following page is shown:
```

Comment:
AFAIK this page is not shown in production build. Am I right?

Reply:
This page is shown in any case, but in the dev build it displays available routes, and in the prod build only a message that the route was not found.